pkg/genericcontrolplane/apiextensions.go

```diff
@@ -87,7 +87,7 @@ func createAPIExtensionsConfig(
 		},
 		ExtraConfig: apiextensionsapiserver.ExtraConfig{
 			CRDRESTOptionsGetter: apiextensionsoptions.NewCRDRESTOptionsGetter(etcdOptions),
-			MasterCount:          3,
+			MasterCount:          1, // TODO: pass this in correctly
```

Comment:
sigh, we really want transactions for config changes for these kind of things, instead of random waits at random places in the system.